recipes-extended/luci/luci_git.bbappend

```diff
@@ -0,0 +1,104 @@
+################################################################################
+#
+# luci_git.bb
```

Comment:
incorrect name, should be bbappend
I think I saw also not correct name inside liblucihttp_git.bbappend

Reply:
fixed